src/main/java/software/amazon/awssdk/crt/http/HttpProxyEnvironmentVariableSetting.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+package software.amazon.awssdk.crt.http;
+
+import software.amazon.awssdk.crt.CrtRuntimeException;
+import software.amazon.awssdk.crt.io.TlsContext;
+import software.amazon.awssdk.crt.http.HttpProxyOptions.HttpProxyConnectionType;
+
+/**
+ * This class provides access to Http proxy environment variable configuration
+ * setting
+ */
+public class HttpProxyEnvironmentVariableSetting {
+
+    private TlsContext tlsContext;
```

Comment:
Why `TlsContext` instead of `TlsConnectionOptions` as what we have in C?

Reply:
Thanks, fixed.